src/app/components/LinkedData/index.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
+import LinkData from '.';
+
+describe('LinkData', () => {
+  const props = {
+    isAmp: false,
+    lang: 'en-GB',
+    seoHeadline: 'Royal wedding 2018: Bouquet laid on tomb of unknown warrior',
+    type: 'article',
+    optimoId: 'urn:bbc:optimo:asset:c9rpqy7pmypo',
```

Comment:
```suggestion
    optimoId: 'c9rpqy7pmypo',
```
This has the snapshot `https://www.bbc.com/news/articles/urn:bbc:optimo:asset:c9rpqy7pmypo`

Reply:
Also affects the snapshot, so applied the fix in a separate commit:

https://github.com/bbc/simorgh/pull/1391/commits/6e596ef0d81b46add48e7ef650b97c74c3852bde